collect.py

```diff
@@ -200,9 +200,21 @@ def do_something(driver, elem_properties=None):
 
 
 def screenshot(driver, file_path):
-    driver.get_screenshot_as_file(file_path)
-    image = Image.open(file_path)
-    image.save(file_path)
+    WINDOW_HEIGHT = 732
+    WINDOW_WIDTH = 412
```

Comment:
The size of the window is not the same on Firefox and Chrome, maybe we should make it the same since now we are removing the scrollbars.
Can you try to make the size the same and collect an example screenshot and post it here?

Reply:
@marco-c I changed size of firefox window from 808 to 732. Size of screenshot changes.
I tested it on previous (808) and it works fine.

Reply:
What are the sizes of the screenshots for Firefox and Chrome before and after modifying the window size?

Reply:
FIREFOX : 412 x 656
CHROME : 412 x 732

Reply:
> FIREFOX : 412 x 656
> CHROME : 412 x 732

This is before or after? (I guess after?)

Reply:
Yeah. Before they both were same and results were fine. So scrolling was same in both (if they starting at same position)

Reply:
Oh, so the inner window size is the same? So we don't need to handle different window sizes here?

Reply:
Yes! They are same. I matched their sizes and how do they scroll for different webpages.